scripts/build-app.mjs

```diff
@@ -12,7 +12,9 @@ async function createApplicationManifest() {
     const manifest = {
         name: pkgConfig.title,
         description: pkgConfig.description,
-        'chromium-args': '--ignore-certificate-errors',
+        // --enable-automation is use to disable password saving bubble
+        // https://github.com/GoogleChrome/chrome-launcher/issues/32
+        'chromium-args': '--ignore-certificate-errors --enable-automation',
```

Comment:
Why is it necessary to disable password popup?
I was eventually planning to use this so users can store their account data and login to websites automatically.

Reply:
Oh ok, sounds like a neat feature. I was just thinking that having this pop up will be annoying but ok